### PR TITLE
Update nanopb to 0.3.3

### PR DESCRIPTION
**Pull request by agatti**

### CODE REVIEW

File: Library/Formula/nanopb-generator.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class NanopbGenerator < Formula
   desc "ANSI C library for encoding and decoding Protocol Buffer messages"
   homepage "http://koti.kapsi.fi/jpa/nanopb/docs/index.html"
-  url "http://koti.kapsi.fi/~jpa/nanopb/download/nanopb-0.2.7.tar.gz"
-  sha256 "acaac15e0a623451b6db0e8c72233bd765b190e7449c1e0ddd9165d3ef41c223"
+  url "http://koti.kapsi.fi/~jpa/nanopb/download/nanopb-0.3.3.tar.gz"
+  sha256 "e8288dcef555c7e06e5e9102a523ece35a79c12e73043f4fe34c05ef73cc3bd2"
   revision 1
```
Comment (bfontaine) [inline Library/Formula/nanopb-generator.rb]: Please remove this line. Revisions are reset when we bump versions.

Comment (agatti) [inline Library/Formula/nanopb-generator.rb]: Taken care of in the subsequent commit.  Thank you for noticing that.

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew @agatti; we appreciate it! :tada:

**Comment by bfontaine** [comment]
Tiny note for the future: the preferred commit message format for version bumps is “`nanopb-generator 0.3.3`”, i.e. “`<name> <version>`”.

**Comment by agatti** [comment]
Point taken, thank you for the hints :)

**Pull request closed without merging**